install/image-references

```diff
@@ -48,3 +48,23 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-cli:latest
+  - name: keepalived-ipfailover
+    from:
+      kind: DockerImage
+      name: registry.svc.ci.openshift.org/openshift:keepalived-ipfailover
+  - name: coredns
+    from:
+      kind: DockerImage
+      name: registry.svc.ci.openshift.org/openshift:coredns
+  - name: mdns-publisher
+    from:
+      kind: DockerImage
+      name: registry.svc.ci.openshift.org/openshift:mdns-publisher
+  - name: haproxy-router
+    from:
+      kind: DockerImage
+      name: registry.svc.ci.openshift.org/openshift:haproxy-router
+  - name: baremetal-runtimecfg
```

Comment:
I raised this on the PR but this name doesn't seem to make sense.  What is the name that is long term stable for this?  What does it own?  As a regular user, how do I know what is in here?

Reply:
Discussed in slack
https://coreos.slack.com/archives/CBN38N3MW/p1563379222233400?thread_ts=1562941823.460800&cid=CBN38N3MW
https://coreos.slack.com/archives/CBN38N3MW/p1563379226233600?thread_ts=1562941823.460800&cid=CBN38N3MW